pages/tutorials/input-files.rst

```diff
@@ -0,0 +1,716 @@
+.. py:currentmodule:: cantera.ctml_writer
```

Comment:
This line, and line 3, are Sphinx-specific things that I think we should probably drop